install-mlx.md

```diff
@@ -8,7 +8,7 @@ In order to use MoonGen/libmoon with Mellanox NICs some addtional steps are nece
 Additional prerequisites
 ------------------------
 
- - Mellanox OFED version: ``4.1``; Download from [here](http://www.mellanox.com/page/products_dyn?product_family=26&mtag=linux). Other versions may work (check the [driver compatibility matrix](http://www.mellanox.com/page/mlnx_ofed_matrix?mtag=linux_sw_drivers)), though that has not been tested.
+ - Mellanox OFED version: ``4.1``; Download from [here](http://www.mellanox.com/page/products_dyn?product_family=26&mtag=linux). Other versions may work (check the [driver compatibility matrix](http://www.mellanox.com/page/mlnx_ofed_matrix?mtag=linux_sw_drivers)), though that has not been tested. On the other hand, the complete ``OFED 4.1`` might no more be installable on current distributions.
```

Comment:
Can you change this to say that it should work with any OFED version but that only 4.1 and 4.4 have been tested properly? Thanks